### PR TITLE
add documentation for object readyState and hideLoadImage

### PR DESCRIPTION
**Pull request by eheckel**
Adds documentation for new object load related API.

- `readyState` property on ObjectHandle
- `readyStateChanged` event on ObjectHandle
- `hideLoadImage` on a SASReportObjectElement

### CODE REVIEW

File: sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md [hunk 1]
```diff
@@ -62,6 +62,12 @@ default value: `true`
 default value: `undefined`<br>
 default behavior: use a shared report context per report
 
+### `hideLoadImage: boolean`
+
+Indicates whether the static loading image displayed during object load should be hidden. When true, the report image is not displayed.
```
Comment (joylashley) [inline sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md:67]: Can "static loading image" change at the beginning of the first sentence? What about this: "Indicates whether the static report image should be hidden when the object is loaded."

File: sdk/va-report-components/documentation/docs/api/ObjectHandle.md [hunk 2]
```diff
@@ -11,6 +11,16 @@ When a report element is assigned new attribute values or removed from the DOM,
 any `ObjectHandles` obtained from that element are invalidated and should be
 discarded.
 
+## Properties
+
+### readyState: string
+Describes the ready state of the report object. When this value changes, a `readyStateChanged` event is fired on the ObjectHandle.
```
Comment (brmorr) [inline sdk/va-report-components/documentation/docs/api/ObjectHandle.md:17]: It seems a little odd to say "Describes" here.  Maybe just `The ready state of the report object.'?

File: sdk/va-report-components/documentation/docs/api/ObjectHandle.md [hunk 3]
```diff
@@ -83,6 +93,7 @@ Adds an event listener to the `ObjectHandle` to call the supplied listener when
 
 `eventType` is a string that represents the event type to listen for. These event types are supported:
 - `"selectionChanged"` for listening for selection changes in the object.
+- `"readyStateChanged"` for listening for changes in the object’s ready state.
```
Comment (brmorr) [inline sdk/va-report-components/documentation/docs/api/ObjectHandle.md:96]:
```suggestion
- `"readyStateChanged"` for listening to changes on the `readyState` property.
```

Comment (brmorr) [reply sdk/va-report-components/documentation/docs/api/ObjectHandle.md:96]: Seems better to directly reference the property that this change event is tracking.

File: sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md (hunk 1 above)
Comment (brmorr) [reply sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md:67]: I was looking at existing doc to see if we have precedence for wording and sentence structure for anything similar.  The closest I can see is the doc for [`restrictViewportGestrues`](https://developer.sas.com/sdk/va/docs/api/SASReportObjectElement/#restrictviewportgestures-boolean). 

It says
> When `true`, report objects that support zooming require a modifier...

So the parallel here might be
> When `false`, report objects display a placeholder static image of the report object while it is loading. `true` hides this placeholder and displays a loading indicator.
> 
> default value: `false`

Comment (joylashley) [reply sdk/va-report-components/documentation/docs/api/SASReportObjectElement.md:67]: That looks good to me, @brmorr. Thanks for finding something similar that we already had.

**Review by brmorr — APPROVED** [review]
LGTM

**Pull request merged**